backup_and_restore/backing-up-etcd.adoc

```diff
@@ -0,0 +1,26 @@
+[id="backup-etcd"]
+= Backing up etcd
+include::modules/common-attributes.adoc[]
+:context: backup-etcd
+
+toc::[]
+
+etcd is the key value store for {product-title}, which persists the state of all
+resource objects.
+
+It is recommended to back up your cluster's etcd data regularly and store in a
```

Comment:
So I wonder about this:

>recommend v
> Avoid if possible. A phrase such as “We recommend that you take the following action” could create a potential marketing or legal problem.

Can we say it's best practice to back up the cluster 's etcd data regularly?